### PR TITLE
Feature/rooms list

### PR DESCRIPTION
**Pull request by paoesco**
Adds a new tab in the menu to display the list of all the rooms that are loaded inside the database.
This is a plain display of room names, remaining tasks will be :
- Search 
- CSS

Also introduces dotenv npm package to load .env file in local development only (in remote "dev" or "production", it uses env variables).
My .env contains only :
MONGODB_URI=mongodb://localhost:27017/wye

### CODE REVIEW

File: routes/rooms.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const router = express.Router();
+const MongoClient = require('mongodb').MongoClient;
+
+const mongoUri = process.env.MONGODB_URI;
+const roomsCollection = "wye-rooms";
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  MongoClient.connect(mongoUri, function(err, db) {
```
Comment (romgar) [inline routes/rooms.js:10]: I guess you planned to move to something like https://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html#mongoclient-connection-pooling for the db initialisation?

Sounds completely fine for a first usage. We will update it in another MR.

File: routes/rooms.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      });
+    });
+    
+
+  });
+});
+
+const findRooms = function(db, callback) {
+  const collection = db.collection(roomsCollection);
+  collection.find({}).toArray(function(err, docs) {
```
Comment (romgar) [inline routes/rooms.js:31]: You don't even need `{}`.

File: routes/rooms.js (hunk 1 above)
Comment (paoesco) [reply routes/rooms.js:10]: Yes but we didn't choose the driver yet so I took one without thinking a lot.
I'll add an issue to be sure to not forget

Comment (romgar) [reply routes/rooms.js:10]: `mongoose` seems nice for the `Schema` feature.

**Pull request merged**